src/diffusers/pipelines/musicldm/pipeline_musicldm.py

```diff
@@ -51,7 +51,7 @@
         >>> import torch
         >>> import scipy
 
-        >>> repo_id = "cvssp/audioldm-s-full-v2"
+        >>> repo_id = "sanchit-gandhi/musicldm-full"
```

Comment:
This is the 'official' checkpoint: https://huggingface.co/ucsd-reach/musicldm
```suggestion
        >>> repo_id = "ucsd-reach/musicldm"
```

Reply:
Thanks, updated with the official checkpoint! :)